recirq/qaoa/gates_and_compilation.py

```diff
@@ -557,28 +558,26 @@ def compile_single_qubit_gates(
         mutate: By default, return a copy of the circuit. Otherwise,
             mutate in place.
     """
-    if mutate:
-        c2 = circuit
-    else:
-        c2 = circuit.copy()
+    assert not mutate, "cirq doesn't mutate."
```

Comment:
Can we piggyback on Cirq's deprecation functionality and deprecate this paramete?
Same below.

Reply:
removed the vestigial arguments entirely